Sources/MixedReality/Microsoft.Psi.MixedReality.UniversalWindows/MediaCaptureMicrophone.cs

```diff
@@ -0,0 +1,244 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Psi.MixedReality
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.Psi;
+    using Microsoft.Psi.Audio;
+    using Microsoft.Psi.Components;
+    using Windows.Foundation;
+    using Windows.Media;
+    using Windows.Media.Capture;
+    using Windows.Media.Capture.Frames;
+    using Windows.Media.MediaProperties;
+    using AudioBuffer = Microsoft.Psi.Audio.AudioBuffer;
+
+    /// <summary>
+    /// Microphone source component with MediaCapture.
+    /// </summary>
+    public class MediaCaptureMicrophone : ISourceComponent, IDisposable
```

Comment:
Since this component is now separate from the camera component, there is only a single audio output stream. To facilitate easier usage with operators, I'd suggest making this component implement `IProducer<AudioBuffer>` and changing the name of the `Emitter` to `Out` instead of `AudioBuffer` (similar to the existing StereoKit-based `Microphone` component.